FarmThread.py

```diff
@@ -94,3 +112,19 @@ def __notifyConnectorDrops(self, newDrops: list):
                     requests.post(self.config.connectorDrops, headers={"Content-type":"application/json"}, json=params)
             else:
                 requests.post(self.config.connectorDrops, json=newDrops)
+
+    def __notifactionToast(self, newDrops: list):
+        if newDrops:
+            for x in range(len(newDrops)):
+                title = newDrops[x]["dropsetTitle"]
+                reward = newDrops[x]["inventory"][0]["localizedInventory"]["title"]["en_US"]
+                toast = win10toast.ToastNotifier()
```

Comment:
Missing change from import
```suggestion
                toast = ToastNotifier()
```